tests/test.js

```diff
@@ -859,6 +870,89 @@ describe('imgix-javascript unit tests', function() {
 		});
 	});
 
+	it('imgix.fluid given an img node as first arg', function() {
```

Comment:
Sorry, last thing--we can probably kill this test, right? Since the `isImage` codepath is gone, it doesn't test anything unique.